wp-content/themes/orient-theme/taxonomy-series.php

```diff
@@ -1,18 +1,35 @@
 <?php
 
-get_header(); ?>
+get_header();
 
+//create a copy of wp_query
+$copy = clone $wp_query;
 
+//alternative???
```

Comment:
Generally I try to avoid committing extraneous code, especially extraneous commented out code -- this includes alternative methods. Every PR should be a completed code change, not a work in progress, so keeping comments like this in just clutters things up (and introduces the possibility that 5 years from now someone reads this and wonders why we didn't go with the alternative.)